### ISSUE TITLE
Update AWS RDS Certificate for ap-northeast-2

### ISSUE THREAD

**Issue by yoonjs2** [felixge/node-mysql#1329]
Hi,

As of Jan 06 2016, AWS Asia Pacific (Seoul) Region (aka. ap-northeast-2) has open.
SSL Certification of AWS RDS has been updated but It's not been applied to node-mysql's cert profile.

https://github.com/felixge/node-mysql/blob/master/lib/protocol/constants/ssl_profiles.js

So my server in new region keep failing after migration from ap-northeast-1 to ap-northeast-2.
Is anyone can help me to use new certificate?

**Comment by dougwilson** [comment]
Hi @yoonjs2 ! That's interesting. I can certainly update them (or you can even make a pul request to update). Can you link me to the public certificates?

**Comment by yoonjs2** [comment]
Thank you @dougwilson 
You can grab updated cert strings at the bottom of https://s3.amazonaws.com/rds-downloads/rds-combined-ca-bundle.pem (line from 261)

```text
-----BEGIN CERTIFICATE-----
MIIEATCCAumgAwIBAgIBTDANBgkqhkiG9w0BAQUFADCBijELMAkGA1UEBhMCVVMx
EzARBgNVBAgMCldhc2hpbmd0b24xEDAOBgNVBAcMB1NlYXR0bGUxIjAgBgNVBAoM
GUFtYXpvbiBXZWIgU2VydmljZXMsIEluYy4xEzARBgNVBAsMCkFtYXpvbiBSRFMx
GzAZBgNVBAMMEkFtYXpvbiBSRFMgUm9vdCBDQTAeFw0xNTExMDYwMDA1NDZaFw0y
MDAzMDUwMDA1NDZaMIGUMQswCQYDVQQGEwJVUzETMBEGA1UECAwKV2FzaGluZ3Rv
bjEQMA4GA1UEBwwHU2VhdHRsZTEiMCAGA1UECgwZQW1hem9uIFdlYiBTZXJ2aWNl
cywgSW5jLjETMBEGA1UECwwKQW1hem9uIFJEUzElMCMGA1UEAwwcQW1hem9uIFJE
UyBhcC1ub3J0aGVhc3QtMiBDQTCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoC
ggEBAKSwd+RVUzTRH0FgnbwoTK8TMm/zMT4+2BvALpAUe6YXbkisg2goycWuuWLg
jOpFBB3GtyvXZnkqi7MkDWUmj1a2kf8l2oLyoaZ+Hm9x/sV+IJzOqPvj1XVUGjP6
yYYnPJmUYqvZeI7fEkIGdFkP2m4/sgsSGsFvpD9FK1bL1Kx2UDpYX0kHTtr18Zm/
1oN6irqWALSmXMDydb8hE0FB2A1VFyeKE6PnoDj/Y5cPHwPPdEi6/3gkDkSaOG30
rWeQfL3pOcKqzbHaWTxMphd0DSL/quZ64Nr+Ly65Q5PRcTrtr55ekOUziuqXwk+o
9QpACMwcJ7ROqOznZTqTzSFVXFECAwEAAaNmMGQwDgYDVR0PAQH/BAQDAgEGMBIG
A1UdEwEB/wQIMAYBAf8CAQAwHQYDVR0OBBYEFM6Nox/QWbhzWVvzoJ/y0kGpNPK+
MB8GA1UdIwQYMBaAFE4C7qw+9hXITO0s9QXBj5yECEmDMA0GCSqGSIb3DQEBBQUA
A4IBAQCTkWBqNvyRf3Y/W21DwFx3oT/AIWrHt0BdGZO34tavummXemTH9LZ/mqv9
aljt6ZuDtf5DEQjdsAwXMsyo03ffnP7doWm8iaF1+Mui77ot0TmTsP/deyGwukvJ
tkxX8bZjDh+EaNauWKr+CYnniNxCQLfFtXYJsfOdVBzK3xNL+Z3ucOQRhr2helWc
CDQgwfhP1+3pRVKqHvWCPC4R3fT7RZHuRmZ38kndv476GxRntejh+ePffif78bFI
3rIZCPBGobrrUMycafSbyXteoGca/kA+/IqrAPlk0pWQ4aEL0yTWN2h2dnjoD7oX
byIuL/g9AGRh97+ssn7D6bDRPTbW
-----END CERTIFICATE-----
```

**Comment by dougwilson** [comment]
Awesome, thanks!

**Comment by dougwilson** [comment]
Ah, nevermind, it's not a bug; looks like they added that certificate around Nov 6, which was after the last time we pulled all the Amazon certificates.

**Comment by yoonjs2** [comment]
@dougwilson Yes, that's true. 
It's not a bug but needed to be updated for the new region.
Would you mind if I create a PR for this? 
(I already made a local update in my server to resolve the problem ASAP)

**Comment by dougwilson** [comment]
Hi @yoonjs2 , if you can, please install the latest from GitHub (instructions https://github.com/felixge/node-mysql#install) and validate that the new certificate is working fro you :)

**Comment by yoonjs2** [comment]
@dougwilson Ah, you've just made fix for this! Thanks!
You've saved my time :)

**Comment by dougwilson** [comment]
Awesome :) Just published to npm as version 2.10.1
